### PR TITLE
KEYCLOAK-9779 Domain mode not supported on Azure

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/KEYCLOAK-9779

### CODE REVIEW

File: server_installation/topics/operating-mode/domain.adoc [hunk 1]
```diff
@@ -38,6 +38,8 @@ many {project_name} server instances will be started on that machine.  When the
 as many {project_name} server instances as it was configured to do.  These server instances pull their configuration
 from the domain controller.
 
+NOTE: In some environments, such as Microsoft Azure, the domain mode is not applicable. Please consult this with {appserver_name} documentation.
```
Comment (slaskawi) [inline server_installation/topics/operating-mode/domain.adoc]: I wanted to provide a [link](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/using_jboss_eap_in_microsoft_azure/index) here but there is no community (Wildfly) equivalent to it. So I think the best idea is to be a bit "enigmatic" here and just ask users to consult it with App Server documentation.

**Comment by slaskawi** [comment]
@andymunro Can we do something with the CI? Both link published by the bot lead to HTTP 404 :(

File: server_installation/topics/operating-mode/domain.adoc (hunk 1 above)
Comment (andymunro) [reply server_installation/topics/operating-mode/domain.adoc]: Yes, I think that is just fine. Thanks for asking, Sebastian.

Comment (andymunro) [reply server_installation/topics/operating-mode/domain.adoc]: Actually, it should be "Please consult the {appserver_name} documentation."

Comment (slaskawi) [reply server_installation/topics/operating-mode/domain.adoc]: Fixed, thanks!

**Comment by slaskawi** [comment]
@hmlnarik @andymunro Could you please take a look at this one?

**Review by andymunro — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

**Pull request merged**